src/django/api/management/commands/resetdb.py

```diff
@@ -22,9 +23,21 @@ def handle(self, *args, **options):
         call_command("reset_schema", "--noinput")
         call_command("migrate")
 
-        # Create test utility located in Raleigh.
-        test_utility = Utility(pwsid="123456789", name="Azavea Test Utility")
-        test_utility.save()
+        # Create test utilities located in Raleigh.
+        test_utility_1 = Utility(pwsid="001001", name="Azavea Test Utility 1")
+        test_utility_1.save()
+
+        test_utility_2 = Utility(pwsid="001002", name="Azavea Test Utility 2")
+        test_utility_2.save()
+
+        test_utility_3 = Utility(pwsid="001003", name="Azavea Test Utility 3")
+        test_utility_3.save()
+
+        test_utility_4 = Utility(pwsid="001004", name="Azavea Test Utility 4")
+        test_utility_4.save()
+
+        test_utility_5 = Utility(pwsid="001005", name="Azavea Test Utility 5")
+        test_utility_5.save()
 
         # Create test users.
```

Comment:
Would you mind adding NC to the validator user? Otherwise the validator user can't access the API.

Reply:
51871e6c36a58752a548b38e9ee417cd9904ab08